interface/forms/newpatient/common.php

```diff
@@ -307,18 +303,6 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
         title='<?php echo xla('Click here to choose a date'); ?>'>
      </td>
     </tr>
-
-    <tr>
-     <td class='text' colspan='2' style='padding-top:1em'>
-<?php if ($GLOBALS['athletic_team']) { ?>
-      <p><i>Click [Add Issue] to add a new issue if:<br />
-      New injury likely to miss &gt; 1 day<br />
-      New significant illness/medical<br />
-      New allergy - only if nil exist</i></p>
-<?php } ?>
-     </td>
-    </tr>
```

Comment:
Yes, be very careful about removing any code not wrapped in this test.  There was surely a reason it was done that way.  Note the colspan.